src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/CollectionGridStyles.kt

```diff
@@ -0,0 +1,39 @@
+package org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid
+
+import javafx.geometry.Pos
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.contentgrid.view.ContentGridStyles
+import tornadofx.*
+
+class CollectionGridStyles: Stylesheet() {
+
+    companion object {
+        val collectionsFlowpane by cssclass()
+        val contentLoadingProgress by cssclass()
+        val innercard by cssclass()
+    }
+
+    init {
+        collectionsFlowpane {
+            vgap = 32.px
+            hgap = 24.px
+            alignment = Pos.TOP_LEFT
+            // Add larger padding on bottom to keep FAB from blocking last row cards
+            padding = box(10.px, 20.px, 95.px, 20.px)
+        }
+
+        contentLoadingProgress {
+            progressColor = AppTheme.colors.appRed
+        }
+        ContentGridStyles.innercard {
+            maxHeight = 118.px
+            maxWidth = 142.px
```

Comment:
Why are you modifying the sizes and stuff here of the cards? I can understand colors, maybe, but sizes?

Reply:
that's been moved out into the widget styles. i just haven't pushed up those changes yet